src/Symfony/Component/Notifier/Bridge/SpotHit/SpotHitTransport.php

```diff
@@ -82,6 +98,8 @@ protected function doSend(MessageInterface $message): SentMessage
                 'type' => 'premium',
                 'message' => $message->getSubject(),
                 'expediteur' => $from,
+                'smslong' => $this->smsLong,
+                'smslongnbr' => $this->smsLongNBr,
```

Comment:
Is it ok to send null here?

Reply:
Here "body" is an RFC1738 query like http_build_query() does. So null values are removed. This case is validated by the tests I added on the basic operation.

https://github.com/symfony/symfony/blob/0773491f79cf2d1555ac83fb5b90139c5d1db9fc/src/Symfony/Component/Notifier/Bridge/SpotHit/Tests/SpotHitTransportTest.php#L57